docs/source/tutorials/datalab/datalab_quickstart.ipynb

```diff
@@ -64,7 +64,7 @@
     "You can use pip to install all packages required for this tutorial as follows:\n",
     "\n",
     "```ipython3\n",
-    "!pip install matplotlib \n",
+    "!pip install matplotlib datasets\n",
```

Comment:
```suggestion
    "!pip install matplotlib \n",
```

Don't need to explicitly state `datasets` as it will be installed with `cleanlab[datalab]`